docs/app/insights.md

```diff
@@ -1,81 +1,199 @@
 # Insights
 
-> This [Module](/getting-started/glossary/#modules) allows the creation of custom
-> [Dashboards](/getting-started/glossary/#dashboards), each with any number of highly configurable
-> [Panels](/getting-started/glossary/#panels) for displaying data.
+> Insights is a module in Directus that allows rapid, no-code buildout of analytics dashboards. Regardless of your
+> use-case (_be it business intelligence, user progress tracking, web traffic conversions, or whatever_!) you will be
+> able to setup key data metrics in just a few clicks.
 
-[[toc]]
+![image of dashboard(s)](../assets/insights-images/insights_dashboards.gif)
 
-## Browsing Dashboards
+- [How it Works](#how-it-works)
+- [Dashboards Overview](#dashboards-overview)
+- [Panels Overview](#panels-overview)
+  - [Panel Header Options](#panel-header-options)
+  - [Label Overview](#label-overview)
+  - [List Overview](#list-overview)
+  - [Metrics Overview](#metrics-overview)
+    - [Style and Format Options](#style-and-format-options)
+  - [Time Series Overview](#time-series-overview)
+  - [Advanced Filtering](#advanced-filtering)
+- [Tutorial Video](#tutorial-video)
+- [Extensibility Options](#extensibility-options)
 
-Clicking into the Insights module immediately presents a listing of all dashboards you have access to. These are shown
-in the navigation bar and a listing on the main page.
+## How it Works
 
-## Creating a Dashboard
+![gif/video of setting up dashboard and panels](image.gif)
 
-1. Click the "Create Dashboard" action button (`+`) in the header.
-2. Fill in the dashboard name, icon and note
-3. Click "Save"
+1. Create a new dashboard by clicking the `+` icon in the upper right hand corner.
```

Comment:
Not sure if it was merged, but we should start using the inline icons that @Nitwel created: `+`

Reply:
Not yet, as we're blocked on figuring out final image specs before that one can be merged 👍🏻

Reply:
okay sweet. "page header". noted.

Reply:
In general, we just want to re-use and reference this naming system... just helps stay consistent: https://docs.directus.io/app/overview/

Reply:
Okay. I will straight-up make flashcards to learn the Directus terms.
Will take time to get it all in my blood though.